applications/nrf_desktop/src/modules/ble_conn_params.c

```diff
@@ -17,6 +17,10 @@
 #include <caf/events/ble_common_event.h>
 #include "ble_event.h"
 
+#ifdef CONFIG_DESKTOP_BLE_MANAGE_CI
```

Comment:
let's add this feature unconditionally and enabled code as needed with IS_ENABLED in source

the justification is that it makes code easier to maintain while optimizer removes the code anyway plus checks for it's correctness even if feature is disabled.
we might pay with slight increase in memory usage at line 55 (increased structure size) but that is cheap (if we want to avoid it we could convert bools to flags but I don't think it's worth it)

Reply:
we have used preprocessor earlier for `CONFIG_CAF_BLE_USE_LLPM` because of some missing defines but we might try to set this code up a bit to reduce complexity.